org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -0,0 +1,94 @@
+package org.code.playground;
+
+import java.io.FileNotFoundException;
+
+public class Board {
```

Comment:
We could add a protected constructor, but not required - that can also be done when we start actually implementing this class.

Reply:
Added!